### ISSUE TITLE
Add Custom Tool on React 

### ISSUE THREAD

**Issue by zackprod** [unlayer/react-email-editor#126]
I want to add my custom tool , but unlayer.create Viewer doesn't work for me.

![customTool](https://user-images.githubusercontent.com/39338983/82753700-9148c080-9dbf-11ea-9ae1-7bf43a15f658.png)

![EmailEditor](https://user-images.githubusercontent.com/39338983/82753699-90b02a00-9dbf-11ea-833e-a66653ca4aea.png)

![error](https://user-images.githubusercontent.com/39338983/82753726-cc4af400-9dbf-11ea-98f3-d75a293f81de.png)

**Comment by grumpycat808** [comment]
I'm getting the exact same issue....

**Comment by adeelraza** [comment]
Custom tools have to be passed in as `customJS` in the `options` prop. Check more on this here: https://docs.unlayer.com/docs/custom-js-css

**Comment by fantypants** [comment]
Has this had any progress? I've been attempting to add a custom tool via customJS; and anyway I have tried to use it, does not work. I have tried using my own scripts, both local & remote, and using the example remote one; all attempts lead to the customJS option almost being ignored.

**Comment by lidrissi** [comment]
> Has this had any progress? I've been attempting to add a custom tool via customJS; and anyway I have tried to use it, does not work. I have tried using my own scripts, both local & remote, and using the example remote one; all attempts lead to the customJS option almost being ignored.

the customJs is a premium feature

**Comment by Alok-jaiswal** [comment]
no, is not a premium feature. If you have use in react, that time you have to put your code in public folder and import in custom.js
like in custom array. Is work fine.

**Comment by AndreyPatseiko** [comment]
[Alok-jaiswal](https://github.com/Alok-jaiswal)Could you provide some details? Looks like for  creation new tools don't have clear instructions.

**Comment by Alok-jaiswal** [comment]
If you have to use locally that time is working fine. But when you use in live url is not work. Then you have to subscribe to unlayer embed editor it free for one custom tool . sing in  and use it . more information is provided to wedsite.

**Comment by jaminur-islam** [comment]
> I want to add my custom tool , but unlayer.create Viewer doesn't work for me.
> 
> ![customTool](https://user-images.githubusercontent.com/39338983/82753700-9148c080-9dbf-11ea-9ae1-7bf43a15f658.png)
> 
> ![EmailEditor](https://user-images.githubusercontent.com/39338983/82753699-90b02a00-9dbf-11ea-833e-a66653ca4aea.png)
> 
> ![error](https://user-images.githubusercontent.com/39338983/82753726-cc4af400-9dbf-11ea-98f3-d75a293f81de.png)

I have  a save problem in nextjs...

**Comment by mastoj** [comment]
@jaminur-islam , did you figure out how to do it when using the wrapper?

**Comment by Prathamesh-Kulkarni01** [comment]
Have you come up with a solution for it?

**Comment by sahilchopra** [comment]
Any one got solution for this problem ?

**Comment by jangamsetty** [comment]
I also got the same issue. To create a custom tool for that  first you have to  login to the Unlayer then register your custom tool with your required properties then use registerTool() method
